android-sdk/src/main/java/org/sagebionetworks/bridge/android/manager/UploadFileConverter.java

```diff
@@ -0,0 +1,23 @@
+package org.sagebionetworks.bridge.android.manager;
+
+import com.squareup.tape.FileObjectQueue;
+
+import java.io.IOException;
+import java.io.OutputStream;
+
+/**
+ * Created by jameskizer on 5/26/17.
+ */
+
+public class UploadFileConverter implements FileObjectQueue.Converter<UploadFile> {
```

Comment:
I see Tape is a new dependency, but it appears unused. James, is there more to this PR, or can these be removed?